clarity/src/vm/analysis/type_checker/tests/mod.rs

```diff
@@ -763,6 +763,66 @@ fn test_unexpected_use_of_field_or_trait_reference() {
     }
 }
 
+#[test]
+fn test_bitwise_good_checks() {
+    let good = [
+        "(>= (+ 1 2 3) (- 1 2))",
```

Comment:
Looks like you have a few test vectors in here that are not bitwise checks.

Reply:
copy-pasta while extracting bitwise to its own tests, removed them now 😄